sdk/core/core/CHANGELOG.md

```diff
@@ -1,5 +1,4 @@
 # Release History
 
-## 1.0.0-dev.1 (Unreleased)
+## 1.0.0-preview.1 (Unreleased)
```

Comment:
Do we publish the "release" artifacts anywhere? And if so, do we need to go and delete the 1.0.0-dev.1 artifacts to avoid sorting issues? I would imagine dev.1 always appears "newer" than preview.1 if sorted alphabetically (or subsequent preview.2, ... preview.X releases).

Reply:
I don't believe we have published anything yet. @danieljurek is there any old docs that need deleting?

Reply:
Maybe https://github.com/Azure/azure-sdk-for-c/releases?
![image](https://user-images.githubusercontent.com/6527137/78942860-656fb680-7a6f-11ea-8107-fac5bcf04fe7.png)

Reply:
Just the release is published. Docs and index have not been published yet. Releases are sorted chronologically by commit date (not by tag) so these will drop off as newer releases come out.